egs/multi_en/s5/conf/ami_ihm/mfcc.conf

```diff
@@ -0,0 +1,4 @@
+--use-energy=false
+--sample-frequency=16000
```

Comment:
Korbinian, I think it's better to use a sox command to downsample the data
in the wav file, rather than messing with the mel-banks high-freq, which
could be harder for users to get right (e.g. also needs to be done in the
mfcc_hires.conf).  Or (even better), downsample them beforehand and dump
them to disk, which might cause less I/O later on.
By messing with the mel high-freq you'd get a different energy than if you
downsampled the signal.

Also, the downsampling beforehand would probably be more efficient if there
are multiple passes of mfcc extraction (which there are).

On Mon, Jun 27, 2016 at 11:05 AM, Korbinian <notifications@github.com>
wrote:

> In egs/multi_en/s5/conf/ami_ihm/mfcc.conf
> <https://github.com/kaldi-asr/kaldi/pull/771#discussion_r68625847>:
>
> > @@ -0,0 +1,4 @@
> > +--use-energy=false
> > +--sample-frequency=16000
>
> WSJ, AMI, ICSI, Tedlium are 16khz, but fisher and switchboard are 8khz.
> Since most applications/research is done on the 8khz part, I'd suggest we
> rather limit the feature banks to 8khz, as opposed to upsampling this data
> to 16khz.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/771/files/d13323180b98a83622b0cb193a3a5cd042468609#r68625847>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADJVu1llR4jOH8czmmR5jrSEIJOWSht-ks5qQBDSgaJpZM4IZ3x7>
> .
>